### PR TITLE
Fixes  #998, Fixes #989

### LINKED ISSUE #989: [Wordpress] Download: Inconsistent font size
Font size for "Download English/Khmer PDF", on Announcement, appears inconsistent in both English and Khmer. 
Link: https://opendevelopmentcambodia.net/announcements/the-temporary-number-of-cambodians-voter-turnout-in-the-commune-election-in-the-fourth-legislature-4-june-2017/#!/story=post-105209
i227 cimgpsh_orig

### LINKED ISSUE #998: Inconsistent font-weight
https://opendevelopmentcambodia.net/?post_type=profiles&p=105176&preview=true

screenshot from 2017-06-08 10-57-21

### PR DESCRIPTION
**Pull request by acorbi**
@Huyeng THis PR contains many changes and might be difficult to review. However I would kindly like to ask you to review it by checking out this branch on PP or on your local machine.

I have been cleaning the CSS and PHP to harmonize the fonts as much as possible, keeping a consistent style-guide for the links on sidebar widgets but also on the instances of ODM Custom posts widget which we use across the sites.

The main thing to consider is that now, all the links exposed by these widgets are ```<h5></h5>``` headings. I have also removed many CSS properties which were basically overriding this, thus making the CSS/LESS files less maintainable.

See some screenshots:

screenshot from 2017-06-12 18-48-53
screenshot from 2017-06-12 18-49-02
screenshot from 2017-06-12 18-49-11
screenshot from 2017-06-12 18-49-21
screenshot from 2017-06-12 18-49-33
screenshot from 2017-06-12 18-49-51

### CODE REVIEW

**Comment by Huyeng** [comment]
It is really good to expose <h5></h5> headings for all the links of the widgets. 
However, adding the bold styling into the headlines which no content doesn't look nice.
There will be a question that why those related links/content need to be highlighted?

Suggestion:
**Have font-weight as normal style for the widget that show only the headline (no content/meta data) and reduce the margin, including:** 

1.
screenshot_1

2.
screenshot_3

3.
screenshot_7

**Have font weight as bold style for the widget that show the headline with contents/meta**
screenshot_4

**Add line height  of content as before because it is hard to read especially in Khmer:**
screenshot_6

**Comment by acorbi** [comment]
@Huyeng Thanks for your comments. I just commited a code change to increase the line-height of the ```<p></p>``` as you suggest.

Regarding the bold style of headings, the important thing here is to keep consistency. We cannot render some headings **bold** and others **normal** since those widgets can  have both content or not...

If we have those headings bold on the homepage, we should have them bold on the widgets. I personally do not think using bold everywhere for those titles is a problem. But if you think it is, which alternative would you suggest?

> There will be a question that why those related links/content need to be highlighted?

Well, those are links to related documents, why not highlight them?

**Comment by acorbi** [comment]
Also, this PR also fixes https://github.com/OpenDevelopmentMekong/wp-odm_theme/issues/989

**Comment by Huyeng** [comment]
Increases the line-height is fine to read in English page, but on Khmer site, it is a bit hard to read because Khmer letters have head and foot like this: សួស្តី (Hello). It is easy to read with 24 line-height. 

It was recalled that regarding the headings bold on widget of the right sidebar or the widget that lists only the headline like that, the team used to request to change the font-weight as normal because it doesn't look good. I can't find that issue right now, maybe it was logged on the old theme. 

Anyways, I still suggest as mentioned earlier. Please keep consistent of these, the only links of headline will be normal, and the headline with some texts should be bold.

**Comment by acorbi** [comment]
> Increases the line-height is fine to read in English page, but on Khmer site, it is a bit hard to read because Khmer letters have head and foot like this: សួស្តី (Hello). It is easy to read with 24 line-height.

I just commited a code change to increase the line-height of the <p></p> as you suggest.

> Anyways, I still suggest as mentioned earlier. Please keep consistent of these, the only links of headline will be normal, and the headline with some texts should be bold.

@Huyeng that does not work. Either we keep all headings **bold** or we keep them **normal** making a distinction of the styling of the headings depending on whether they have contents or not is not consistant because it could be that the first item has content and the second has not, and then they will be styled differently.

An alternative that I can think of would be to style ALL the headings on sidebar-widgets **normal** and those on other parts of the system (i.e homepage) bold. But this is not the optimal solution, in my opinion.

**Comment by acorbi** [comment]
@Huyeng I understand what you mean. But it seems that you are forgetting that those widgets can be configured to show content or not. See following screenshots:

The widgets can get configured to show different fields (description, date, resources, etc...)
screenshot from 2017-06-13 15-01-17

On https://opendevelopmentmekong.net/topics/land the widgets show a date underneath the title
screenshot from 2017-06-13 14-59-23

Whereas the same widgets on ODC just show the title:
screenshot from 2017-06-13 15-00-28

So this means that depending what is configured items can show content underneath. How would you then implement the logic you are suggesting? and most important, how would you avoid that inconsistencies appear if editors change that configuration.

Keeping consistency involves introducing as less exceptions to a rule as possible.

**Comment by Huyeng** [comment]
I understood now, why the links of related contents were changed to be bold. Before, I have no answer for my team. 

Seeing these, it was demonstrated that the related content displaying on ODC and ODM are not consistent now.  
For ODM related content of land, it will look fine with the bold header of title as it has the date below. 
For ODC, there is no date below the title, if bold the headline, it doesn't look good. Are bold texts easy read? Yes, it may be fine in English but not Khmer. From ODC team, we still would like have the normal texts for links of headlines. 

What do you think if I would like to suggest that provide an option on the widget to show bold or normal of headline, then it will be easy to manage them.

**Comment by acorbi** [comment]
> What do you think if I would like to suggest that provide an option on the widget to show bold or normal of headline, then it will be easy to manage them.

I do not agree with doing this. It is introducing more configuration elements and we should simplify!!

Let's do the following:

1. headings on sidebar widgets will ALL have **normal** styling
2. heading on other places will ALL have **bold** styling

Do you agree with this? I do not have other alternatives to offer you at this point

**Comment by Huyeng** [comment]
What do you refer to the "sidebar widgets"?

**Comment by acorbi** [comment]
Sidebar widget are those **widgets** that are placed on the right **sidebar**. These are present on:

topic pages
https://pp.opendevelopmentmekong.net/topics/land/

screenshot from 2017-06-13 15-48-05
screenshot from 2017-06-13 15-48-14

profile pages
https://pp.opendevelopmentcambodia.net/profiles/access-to-public-service/
screenshot from 2017-06-13 15-50-09

single news pages
screenshot from 2017-06-13 15-51-57

**Comment by acorbi** [comment]
@Huyeng another alternative I can think of is that we style ALL headings as **normal**, EVERYWHERE.

Is this a possibility for your?

**Comment by Huyeng** [comment]
It is ok, If the sidebar widgets are referred to those widgets on the topic pages, profile and its subpages, the single page of news, announcement and detail page of detaset.  
However, home page and news archive page should has the bold headline, otherwise, it is not distinguish between the headline and description because they has some texts below.

**Comment by acorbi** [comment]
@Huyeng OK, we have an agreement. I will make the change and let you review/merge or comment if you want to change something else.

**Comment by Huyeng** [comment]
💯 Cheer! We reached the agreement now.

**Comment by acorbi** [comment]
:clap:

File: inc/templates/post-list-single-1-cols.php [hunk 1]
```diff
@@ -34,10 +34,10 @@
 						endif; ?>
 					<?php echo $localized_title; ?>
 				</a>
-			</h4>
+			</h5>
 		<?php else: ?>
 			<p>
```
Comment (Huyeng) [inline inc/templates/post-list-single-1-cols.php]: Why is <p> tag added above the H5?

Comment (Huyeng) [inline inc/templates/post-list-single-1-cols.php]: This is one also?

**Pull request merged**